pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+# PHEP Applicability
+**Standards** Track PHEPs are recommended standards for PyHC packages. Most packages should make a good-faith effort to comply. PyHC core packages are strongly recommended to comply and generally should do so, absent a compelling reason not to. A PHEP which core packages cannot agree to implement likely does not have sufficient community support for approval. Standards track PHEP compliance should be considered as a factor in evaluating current and potential PyHC packages.
+
+**Informational** PHEPs are for the benefit of PyHC developers and users and may represent "best practices". They are not binding.
+
+**Process** PHEPs determine the process by which the PyHC community makes formal decisions and as such bind all participants in PyHC.
+
+For all three kinds, key words in PHEPs must be interpreted according to [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119); the use of `must`, `must not`, `should`, and `should not` are preferred to the alternatives.
+
+# PHEP Workflow
+## PHEP Editors
+<a name="phep-editors"></a>
+The PHEP editors are individuals responsible for managing the administrative and editorial aspects of the PHEP workflow (e.g. assigning PHEP numbers and changing their status). See [PHEP Editor Responsibilities & Workflow](#phep-editor-responsibilities-workflow) for details.
+
+PHEP editorship is by nomination of the current editors, subject to approval of PyHC leadership. In the event there are fewer than two active editors, PyHC leadership may appoint editors from active participants in the community, sufficient to bring the total to two.
+
+The editors can be contacted by mentioning `@heliophysicsPy/pep-editors` on GitHub. All of the PHEP workflow is conducted via the GitHub PyHC standards repository issues and pull requests.
+
+## Start with an idea
+The PHEP process begins with a new idea for PyHC. It is highly recommended that a single PHEP contain a single key proposal or new idea; the more focused the PHEP, the more successful it tends to be. The PHEP editors reserve the right to reject PHEP proposals if they appear too unfocused or too broad. If in doubt, split your PHEP into several well-focused ones.
+
+Each PHEP must have a champion - someone who writes the PHEP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The PHEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is PHEP-able. Posting to the PyHC mailing list or Element/Slack is usually the best way to go about this.
```

Comment:
```suggestion
Each PHEP must have a champion — someone who writes the PHEP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The PHEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is PHEP-able. Posting to the PyHC mailing list or Element/Slack is usually the best way to go about this.
```
Using an em dash here because I'm addicted to unicode characters.  Also, love the term "PHEP-able"!

Reply:
I can't take credit for PHEP-able; it's from PEP 1 (PEP-able). See Unicode discussion below :)